cpp/include/milvus-storage/packed/reader.h

```diff
@@ -95,19 +72,19 @@ class PackedRecordBatchReader : public arrow::RecordBatchReader {
   private:
   // Advance buffer to fill the expected buffer size
   arrow::Status advanceBuffer();
+  std::vector<const arrow::Array*> collectChunks(int64_t chunksize) const;
```

Comment:
Do we have an implementation for this function?

Reply:
it is implemented in chunk_manager.cpp